<launch>
    <arg name="depends_only" default="false" />
    <arg name="use_game_settings" default="false"/>

    <rosparam command="load" file="$(find bitbots_head_behavior)/config/head_config.yaml"/>

    <group if="$(arg use_game_settings)">
        <rosparam command="load" file="$(find bitbots_bringup)/config/game_settings.yaml" />
    </group>

    <node name="bio_ik_service" pkg="bio_ik_service" type="bio_ik_service"  output="screen"/>

    <node unless="$(arg depends_only)" name="head_behavior" pkg="bitbots_head_behavior" type="head_node.py"  output="screen"/>
</launch>
